i18n/extract.py

```diff
@@ -65,6 +65,22 @@ def add_args(self):
         Adds arguments
         """
         self.parser.description = __doc__
+        self.parser.add_argument(
+            '--merge-js',
```

Comment:
I'm not sure how I feel about this name. It's merging js *and* non-js, maybe `--merge-po` or `--merge-translations`? I don't love those suggestions either.

Reply:
I think `--merge-into=django.po`, `--single-po-file`, `--merge-po-files` or `--single-file` would make sense as well. 

My preference is `--merge-po-files`. It's straightforward.

Reply:
I've used `--merge-po-files`

Reply:
@brian-smith-tcril would you mind taking another look?